odometry_fusion:
  pose_covariance: 
    [
      0.0001, 0.0, 0.0, 0.0, 0.0, 0.0,
      0.0, 0.0001, 0.0, 0.0, 0.0, 0.0,
      0.0, 0.0, 0.000001, 0.0, 0.0, 0.0,
      0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
      0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
      0.0, 0.0, 0.0, 0.0, 0.0, 0.000001
    ]
  twist_covariance: 
    [
      0.00001, 0.0, 0.0, 0.0, 0.0, 0.0,
      0.0, 0.00001, 0.0, 0.0, 0.0, 0.0,
      0.0, 0.0, 0.0001, 0.0, 0.0, 0.0,
      0.0, 0.0, 0.0, 1.0, 0.0, 0.0,
      0.0, 0.0, 0.0, 0.0, 1.0, 0.0,
      0.0, 0.0, 0.0, 0.0, 0.0, 0.00001
    ]
